components/GoldSponsors.js

```diff
@@ -5,7 +5,7 @@ export default function GoldSponsors({ className = '', showSupportBanner = true
             <a
             href="https://www.redhat.com/"
             target="_blank"
-            className="block relative text-center w-2/3 px-14 py-4 sm:p-0 sm:w-1/2 md:w-1/3 lg:w-1/5"
```

Comment:
Instead of removing `px-14`, replace it with `px-4` there for better image size in mobile view. Right now, images look very big in the mobile view as compared to the Headings of the website. Kindly do it for all `px-14`.

Reply:
> Instead of removing `px-14`, replace it with `px-4` there for better image size in mobile view. Right now, images look very big in the mobile view as compared to the Headings of the website. Kindly do it for all `px-14`.

@akshatnema as per suggestions all changes have been made.